cmake/environmentchecks.cmake

```diff
@@ -55,6 +55,26 @@ mark_as_advanced(
   VALGRIND_EXE
 )
 
+# Use clcache/ccache gets used, if available.
```

Comment:
\# If available and wanted use clcache/ccache.

Reply:
Done in commit db1b5cc.